### ISSUE TITLE
Scaffold new Neutrino projects with a CLI helper

### ISSUE THREAD

**Issue by eliperelman** [mozilla-neutrino/neutrino-dev#360]
I foresee enabling a very minimal scaffolding tool that could work with yarn or npx:

```bash
# yarn
yarn create neutrino-project my-project

# npx
npx create-neutrino-project my-project
```

This would then open an interactive session to select project type, like React, Node.js, Web, React components, or a library, maybe to add linting, and then testing. This would output a new directory where we could:

- `npm init` or `yarn init`, depending on which command was used to start the scaffold.
- `git init` with a `.gitignore`
- add a `.neutrinorc.js` file already configured with the presets/middleware
- add scripts to `start`, `build`, `lint`, `test`
- Add a `src` directory with super minimal `index.js`
- If tests are set up, we could generate a test dir
- If linting is set up, we could also put a `.eslintrc.js` file in there too

**Comment by eliperelman** [comment]
This would let us eliminate maintaining the starter kits, and allow for a much quicker developer experience, as well as easier documentation for getting started.

**Comment by helfi92** [comment]
I agree, maintaining the starter kits is kind of a hassle. I love this idea. I can work on it!

**Comment by eliperelman** [comment]
Awesome! I have reserved the package name in the meantime so we can publish it when ready. Feel free to add another monorepo directory for this.

**Comment by helfi92** [comment]
I have an initial patch for this: https://github.com/mozilla-neutrino/neutrino-dev/pull/376

**Comment by helfi92** [comment]
I am hesitant to perform a `git init` as well as adding a `.gitignore` file. This assumes the user is using git as their version control system. If I was using bitbucket for example, I would probably be bothered with all the git generated files. Let me know if you have a different opinion.

**Comment by helfi92** [comment]
@eliperelman Are there any other features we would like to add to the scaffolding tool besides the ones that got merged? I can help out.

**Comment by eliperelman** [comment]
Vue and preact would be epic.

**Comment by helfi92** [comment]
Oh yeah! Awesome idea. I'll look into it.

**Comment by eliperelman** [comment]
We should probably merge into master now so that way we can easily add some new packages for vue and preact.

**Comment by helfi92** [comment]
Sounds good to me. Feel free to perform the merge :)

**Comment by eliperelman** [comment]
Addressed in #376.
